.travis.yml

```diff
@@ -1,27 +1,72 @@
-language: go
 sudo: true #for docker service to be enabled
 os: linux
+language: go
+go: [1.9.2]
+cache:
+  directories:
+    - vendor
+
+
+stages:
+  - dep-cache
+  - test
+  - name: build
+    if: (type != pull_request AND branch = master AND tag IS blank) OR tag IS present
+  - name: release
+    # we are not ready for deployment yet - gomobile depends on xcode, and we didn't test it yet on xgo image
+    #if: tag IS present
+    if: type = never
+
+
+jobs:
+  include:
+    # Dependencies cache stage
+    - stage: dep-cache
+      name: "Dependencies update"
+      script: make deps
+
+    # Test stage
+    - stage: test
+      script: scripts/xgo_run.sh make test
+      name: "Unit tests"
+
+    # Build artifacts
+    - stage: build
+      script:
+      - go get github.com/karalabe/xgo
```

Comment:
Why not use our own: https://github.com/mysteriumnetwork/xgo ?

Reply:
I believe that it's better to use an outside project if it's possible because it's maintainable.

Reply:
Why I agree that it's better to use an outside project, I have a feeling this was forked for a reason.